src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -555,7 +555,7 @@ public static LogisticRegressionBinaryTrainer LogisticRegression(this BinaryClas
         /// <param name="l1Regularization">The L1 <a href='tmpurl_regularization'>regularization</a> hyperparameter. Higher values will tend to lead to more sparse model.</param>
         /// <param name="l2Regularization">The L2 weight for <a href='tmpurl_regularization'>regularization</a>.</param>
         /// <param name="optimizationTolerance">Threshold for optimizer convergence.</param>
-        /// <param name="historySize">Memory size for <see cref="Microsoft.ML.Trainers.PoissonRegressionTrainer"/>. Low=faster, less accurate.</param>
+        /// <param name="historySize">Number of previous iterations to remember for estimate of Hessian. Lower values mean faster but less accurate estimates.</param>
```

Comment:
>for estima [](start = 80, length = 10)

for the estimate of the Hessian.

Reply:
done

---
In reply to: [268038027](https://github.com/dotnet/machinelearning/pull/3059#discussion_r268038027) [](ancestors = 268038027)